spec/features/purchasing_a_reservation_on_behalf_of_spec.rb

```diff
@@ -1,17 +1,18 @@
 require "rails_helper"
 
 RSpec.describe "Purchasing a reservation on behalf of another user" do
-  fixtures :all
 
-  let(:facility) { facilities(:facility) }
-  let(:instrument) { products(:reservation_only_instrument) }
-  let(:facility_admin) { users(:facility_admin) }
-  let(:user) { users(:normal_user) }
+  let!(:instrument) { FactoryGirl.create(:setup_instrument) }
+  let!(:facility) { instrument.facility }
+  let!(:account) { FactoryGirl.create(:nufs_account, :with_account_owner, owner: user) }
+  let!(:price_policy) { FactoryGirl.create(:instrument_price_policy, price_group: PriceGroup.base.first, product: instrument) }
```

Comment:
The price group might need to change for DC, or likely, we'll need to add the user or account explicitly into a price group (UIC doesn't have user-based price groups).